trinity/utils/logging.py

```diff
@@ -19,8 +19,10 @@ def setup_trinity_logging(level: int) -> Tuple[Logger, Queue, handlers.QueueList
 
     handler = logging.StreamHandler(sys.stdout)
 
+    # TODO: allow configuring `detailed` logging
```

Comment:
Just a point of note.  We could allow both of these values `fmt` and `datefmt` to be set via environment variable.  Near zero overhead on our part in terms of code we have to maintain and a small bit of documentation to expose this low level feature.